apps/codeforafrica/src/components/Input/Input.test.js

```diff
@@ -0,0 +1,16 @@
+import { createRender } from "@commons-ui/testing-library";
+import React from "react";
+
+import Input from "./Input";
```

Comment:
This works but for consistency, wouldn't it be better to always import from `@/codeforafrica/...`?

Reply:
When you're importing from same folder, this is still the recommended approach. Issues come when you're importing from other folders relative a give file e.g. `import X from "../../components/Y";`. This is where aliasing shine @esirK .